h2o-algos/src/main/java/hex/glm/GLM.java

```diff
@@ -2319,6 +2323,10 @@ protected void updateProgress(boolean canScore){
       if(canScore && (_parms._score_each_iteration || timeSinceLastScoring() > _scoringInterval)) {
         _model.update(_state.expandBeta(_state.beta()), -1, -1, _state._iter);
         scoreAndUpdateModel();
+        if (!_earlyStop)
```

Comment:
simplify same as above _earlyStop = _earlyStop || ScoreKeeper...
also, this is the exact same code as above, so even better move to  a `updateEarlyStop()` method

Reply:
Yeah, good idea.